sdk/table/azure-table/azure/table/_table_service_client.py

```diff
@@ -0,0 +1,279 @@
+            cors=cors
+        )
+        try:
+            return self._client.service.set_properties(props, **kwargs)  # type: ignore
+        except HttpResponseError as error:
+            process_storage_error(error)
+
+    @distributed_trace
+    def create_table(
+            self,
+            table_name,
+            headers=None,
+            **kwargs
+    ):
+        # type: (...) -> TableClient
+        """Creates a new table under the given account.
+
+                :param headers:
+                :param table_name: The Table name.
+                :type table_name: ~azure.table._models.Table
+                :return: TableClient, or the result of cls(response)
+                :rtype: ~azure.table.TableClient or None
+                :raises: ~azure.core.exceptions.HttpResponseError
+                """
+        table_properties = TableProperties(table_name=table_name, **dict(kwargs, headers=headers))
+        self._client.table.create(table_properties)
+        table = self.get_table_client(table=table_name)
+        return table
+
+    @distributed_trace
+    def delete_table(
+            self,
+            table_name,
+            request_id_parameter=None,
+            **kwargs
+    ):
+        # type: (...) -> None
+        """Creates a new table under the given account.
+
+                        :param request_id_parameter: Request Id parameter
+                        :type request_id_parameter: str
+                        :param table_name: The Table name.
+                        :type table_name: str
+                        :keyword callable cls: A custom type or function that will be passed the direct response
+                        :return: None
+                        :rtype: ~None
+                        """
+        response = self._client.table.delete(table=table_name, request_id_parameter=request_id_parameter, **kwargs)
+        return response
+        # table = self.get_table_client(table=table_name)
+        # table.delete_queue(table_name)
+
+    @distributed_trace
+    def list_tables(
+            self,
+            query_options=None,  # type: Optional[QueryOptions]
+            headers=None,
+            **kwargs
+    ):
+        # type: (...) -> ItemPaged
+        """Queries tables under the given account.
+
+        :param query_options: Parameter group.
+        :type query_options: ~azure.table.models.QueryOptions
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: ItemPaged, or the result of cls(response)
+        :rtype: ~ItemPaged
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        command = functools.partial(
+            self._client.table.query,
+            **dict(kwargs, headers=headers))
+        return ItemPaged(
+            command, results_per_page=query_options,
+            page_iterator_class=TablePropertiesPaged
+        )
+
+    @distributed_trace
+    def query_tables(
```

Comment:
Do query_tables and list_tables return the same thing? Ignore if this is still a design decision to be made